python/ray/ml/config.py

```diff
@@ -145,4 +148,5 @@ class RunConfig:
     name: Optional[str] = None
     local_dir: Optional[str] = None
     callbacks: Optional[List[Callback]] = None
+    stop: Optional[Union[Mapping, Stopper, Callable[[str, Mapping], bool]]] = None
```

Comment:
Yeah I know. Just copied the type def from `tune.run()`...

Reply:
I only see mapping in the following examples.

Reply:
As I mentioned, I just copied these from `tune.run` parameter and these are all options supported by it.